src/rt/mod.rs

```diff
@@ -0,0 +1,16 @@
+//! Runtime components
+//!
+//! By default, hyper includes the [tokio](https://tokio.rs) runtime.
+//!
+//! If the `runtime` feature is disabled, the types in this module can be used
+//! to plug in other runtimes.
+
+pub mod timer;
```

Comment:
Perhaps let's start with the module being private, we can always make it public later if it'd be helpful.